spring-xsuaa/README.md

```diff
@@ -1,4 +1,9 @@
-# SAP BTP Spring XSUAA Security Library 
+### :warning: Deprecation Notice
+In alignment with the end of support for [Spring Boot 2.x](https://spring.io/projects/spring-boot#support) and [Spring Framework 5.x](https://spring.io/projects/spring-framework#support), we will also discontinue support for this library by the **end of 2025** and for the corresponding Spring Boot library [spring-xsuaa-starter](../spring-xsuaa-starter) by **Nov, 2023**.
```

Comment:
Suggestion: "[...] discontinue support for **version 2.x of this library** by [...]"